### ISSUE TITLE
[Bug]: Automatic merge failed when file mode changes

### ISSUE THREAD

**Issue by Fs02** [AndreasAugustin/actions-template-sync#380]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

"Pull template" failed when there's conflict with file mode between source and target repository

### Expected Behavior

template repository file mode should be used

### Current Behavior

it errored and stopped the job, the last message is:

```
Automatic merge failed; fix conflicts and then commit the result.
```

### Template sync version Version

1.1.1

**Comment by AndreasAugustin** [comment]
Hi @Fs02 and thank you for the bug report.
I did not find time yet to look into it but it sounds interesting.
Think a fast  option could be to use `git config --global core.fileMode false` but reading your description you would rather like to use the file mode within the source (template) repository.

Will think about a solution but I guess it will take some days.

**Comment by AndreasAugustin** [comment]
@all-contributors please add @Fs02  for bug

**Comment by AndreasAugustin** [comment]
@Fs02 sorry was busy. Today I was able to confirm the bug with a small test.
A fix will be provided soon.

**Comment by AndreasAugustin** [comment]
@Fs02 a new release is available `v1.1.3`. Hopefully this will solve your issue.
